Source/DafnyCore/AST/Formatting.dfy

```diff
@@ -0,0 +1,97 @@
+/// Microsoft.Dafny.Formatting.ReindentProgramFromFirstToken() takes
+/// - A first token, from which an entire program is available
+/// - A IIndentationFormatter, that knows how to indent every token
+/// It then prints every token (proven) in order, reindenting its leading trivia and trailing trivia
+
+module {:extern "System"} {:compile false} {:options "-functionSyntax:4"} System {
```

Comment:
Should this external module be in a separate file?

Reply:
> I'm a bit confused because the Dafny code doesn't seem to be used. It's just externals right?

Yikes ! Let me know what happened during this refactoring.

This external module acts as an "include" with an equivalent of "use", so I think it's relevant to keep it in this file, until we actually need to reuse it.